### PR TITLE
Implement "Upgrading a SAP HANA Cluster"

### PR DESCRIPTION
**Pull request by tomschr**
This PR implements FATE#320368:

* add chapter with sections taken from Diego's blog article
* include chapter in main file

### CODE REVIEW

File: xml/s4s_upgrade_sap_hana_cluster.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  </procedure>
+
+ <sect1 xml:id="sec-upgrade-sap-hana-cluster-preparing">
+  <title>Preparing the Upgrade</title>
+  <para>
+   Start this task on the secondary node:
+  </para>
+  <procedure>
+   <step>
+    <para>Install the <package>yast2-hana-update</package> package:</para>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/s4s_upgrade_sap_hana_cluster.xml]: Ahh, good to know, thanks! In that case, I need to change a bit more: 

* the introductory paragraph needs to be moved ("start this task on the secondary node") to step 2
* insert the "on both nodes" on your marked line.
* Insert "On the secondary node, start the YaST Control Center and call the "SUSE HANA Cluster Update" module."

**Comment by tomschr** [comment]
> Maybe you could include the link to the blog post somewhere, so the users can also see the screenshots.

I was thinking about that too. :wink: Normally, I would add our standard phrase "Refer to ABC for more information." However, I'm not sure if that applies here. The information is basically the same, apart from the screenshots. Sending the customer to a different spot just for the screenshots sounds also a bit strange. So I'm looking for a catchy phrase which doesn't give the wrong impression.

The best what I came up was: "Refer to the SUSE blog post .... if you would like to comment." Not good, but not wrong.

Any better idea?

**Comment by diegoakechi** [comment]
@tomschr Any special reason to not include the screenshots on the documentation? For me, that would be the best option, then I could change my blog to refer to the official documentation.

**Comment by tomschr** [comment]
> Any special reason to not include the screenshots on the documentation?

@diegoakechi Yes, I was a bit hesitant to add screenshots as they can become of a maintenance burden. It can be difficult to replicate just this specific scenario. Or the GUI has been changed and uses d different theme, wording etc. (happened to all of us). 

I'm not saying we shouldn't include them. If we include them, it could mean that I will bother you in the future again. :wink:

**Comment by diegoakechi** [comment]
@tomschr I see... I will leave the decision to you, as I am ok with both solutions, but don't hesitate to bother me to get things done. :wink:

**Comment by tomschr** [comment]
@diegoakechi
> I see... I will leave the decision to you, as I am ok with both solutions, 

Ok, in that case I would prefer to leave it as is for the time being.


> but don't hesitate to bother me to get things done. wink

Ok, will do that. :wink: 

Thanks for the review, much appreciated! :+1:

**Pull request merged**